monitoring/base/victoriametrics/rules/monitoring-scrape.yaml

```
@@ -0,0 +1,134 @@
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMPodScrape
+metadata:
+  name: victoriametrics-operator
+  namespace: monitoring
+  labels:
+    smallset: "true"
+spec:
+  namespaceSelector:
+    matchNames: [monitoring]
+  selector:
+    matchLabels:
+      control-plane: victoriametrics-operator
+  podMetricsEndpoints:
+  - port: http
+    relabelConfigs:
+      - replacement: victoriametrics-operator
+        targetLabel: job
+      - sourceLabels: [__meta_kubernetes_pod_container_name]
+        regex: manager
+        action: keep
+      - sourceLabels: [__address__]
+        action: replace
+        regex: ([^:]+)(?::\d+)?
+        replacement: ${1}
+        targetLabel: instance
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMServiceScrape
+metadata:
+  name: vmalertmanager
+  namespace: monitoring
+  labels:
+    smallset: "true"
+spec:
+  namespaceSelector:
+    matchNames: [monitoring]
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: vmalertmanager
+      app.kubernetes.io/name: vmalertmanager
+      managed-by: vm-operator
+  endpoints:
+  - port: web
+    relabelConfigs:
+      - replacement: vmalertmanager
+        targetLabel: job
+      - sourceLabels: [__address__]
+        action: replace
+        regex: ([^:]+)(?::\d+)?
+        replacement: ${1}
+        targetLabel: instance
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMPodScrape
+metadata:
+  name: vmagent-smallset
+  namespace: monitoring
+  labels:
+    smallset: "true"
+spec:
+  namespaceSelector:
+    matchNames: [monitoring]
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: vmagent-smallset
+      app.kubernetes.io/name: vmagent
+      managed-by: vm-operator
+  podMetricsEndpoints:
+  - relabelConfigs:
+      - replacement: vmagent-smallset
+        targetLabel: job
+      - sourceLabels: [__meta_kubernetes_pod_container_name]
+        regex: vmagent
+        action: keep
+      - sourceLabels: [__address__]
+        action: replace
+        regex: ([^:]+)(?::\d+)?
+        replacement: ${1}
+        targetLabel: instance
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMPodScrape
+metadata:
+  name: vmalert-smallset
+  namespace: monitoring
+  labels:
+    smallset: "true"
+spec:
+  namespaceSelector:
+    matchNames: [monitoring]
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: vmalert
+      managed-by: vm-operator
+      vmalert-smallset-cluster: "true"
```

I'm a little confused. If we use two sets of vmalert(`vmalert-smallset-0`, `vmalert-smallset-1`) and they work independently, it is better to use the following labels.

```suggestion
      app.kubernetes.io/instance: vmalert-smallset-0
      app.kubernetes.io/name: vmalert
      managed-by: vm-operator
```